### PR TITLE
Fix bug: copy to a replicated table from an empty file (7X)

### PR DESCRIPTION
**Pull request by hyongtao-db**
Long log:

This pr can fix bug reported in https://github.com/greenplum-db/gpdb/issues/16250
The bug is introduced by https://github.com/greenplum-db/gpdb/pull/14989, which forgets to check the corner test case.
Solution: 	
set the default value of `first_segment_rows_completed` and `first_segment_rows_rejected` as 0

Signed-off-by: Yongtao Huang <yongtaoh@vmware.com>

### CODE REVIEW

**Review by higuoxing — APPROVED** [review]
LGTM!

File: src/backend/cdb/cdbcopy.c [hunk 1]
```diff
@@ -644,7 +644,7 @@ cdbCopyEndInternal(CdbCopy *c, char *abort_msg,
 			}
 
 			/* in SREH mode, check if this seg rejected (how many) rows */
-			if (res->numRejected > 0)
```
Comment (kainwen) [inline src/backend/cdb/cdbcopy.c:647]: Please add some comments here.

File: src/test/regress/input/gpcopy.source [hunk 2]
```diff
@@ -1376,3 +1376,16 @@ COPY issue_14353_multi_dist FROM '/tmp/issue_14353_multi_dist_<SEGID>.csv' WITH
 -- Test that gpdb emits an error message saying the data distribution is malformed.
 \! cp /tmp/issue_14353_multi_dist_1.csv /tmp/issue_14353_multi_dist_0.csv
 COPY issue_14353_multi_dist FROM '/tmp/issue_14353_multi_dist_<SEGID>.csv' WITH CSV DELIMITER ',' ON SEGMENT;
+
+-- special case about distributed replicated table
```
Comment (kainwen) [inline src/test/regress/input/gpcopy.source:1380]: Mention Github Issue here.

File: src/backend/cdb/cdbcopy.c (hunk 1 above)
Comment (hyongtao-db) [reply src/backend/cdb/cdbcopy.c:647]: OK

File: src/test/regress/input/gpcopy.source (hunk 2 above)
Comment (hyongtao-db) [reply src/test/regress/input/gpcopy.source:1380]: OK

**Comment by zhaorui20005** [comment]
Maybe you can set both the initial values of `first_segment_rows_completed ` and `first_segment_rows_rejected ` to 0 to solve this issue. I think it looks easier to be understood.

**Comment by hyongtao-db** [comment]
> Maybe you can set both the initial values of `first_segment_rows_completed ` and `first_segment_rows_rejected ` to 0 to solve this issue. I think it looks easier to be understood.

Cool. I think it is a better way. Thanks a lot. I will do it right now.

**Review by zhaorui20005 — APPROVED** [review]
LGTM

**Review by huiliang-liu — APPROVED** [review]

**Comment by hyongtao-db** [comment]
CI pipeline turns green: https://dev.ci.gpdb.pivotal.io/teams/main/pipelines/gpdb-dev-hyongtao-rocky8

**Pull request merged**